tasks/libvirt_default_net_ipv6_tasks.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Parse libvirt default network configuration
+  virt_net:
+    command: get_xml
+    name: default
+  register: default_net_xml
+- debug: var=default_net_xml.stdout
+- name: Remove IPv4 configuration
+  xml:
+    xmlstring: "{{ default_net_xml.get_xml }}"
+    xpath: /network/ip
+    state: absent
+  register: editednet_noipv4
+- name: Configure it as an isolated network
+  xml:
+    xmlstring: "{{ editednet_noipv4.xmlstring }}"
+    xpath: /network/forward
+    state: absent
+  register: editednet_isolated
+- name: Edit libvirt default network configuration, 1
```

Comment:
instead of naming the tasks with numbers -  1,2,3 ... I would name them regrading the attribute we are changing - address, prefix and so on

Reply:
it makes sense